packages/yoroi-extension/app/ergo-connector/stores/ConnectorStore.js

```diff
@@ -232,23 +232,24 @@ export default class ConnectorStore extends Store<StoresMap, ActionsMap> {
     try {
       const wallets = await getWallets({ db: persistentDb });
 
-      const ergoWallets = wallets.filter(
-        wallet => isErgo(wallet.getParent().getNetworkInfo())
-      );
+      const filteredWallets = wallets
+      // list all wallet for now until we find a way to
+      // know which type of wallet to filter ( ergo || cardano )
+      // .filter(wallet => !isErgo(wallet.getParent().getNetworkInfo()));
```

Comment:
@AhmedIbrahim336 , let's plz rework this part here. In the `inject.js` in the connector, depending on which function have been called: `ergo_request_read_access` or `cardano_request_read_access` - different messages are sent: `"connector_connect_request/ergo"` or `"connector_connect_request/cardano"`.

Let's change these messages to instead include the `protocol` field, like:
```
window.postMessage({
    type: "connector_connect_request",
    protocol: "ergo",
}, location.origin);```

or 

```
window.postMessage({
    type: "connector_connect_request",
    protocol: "cardano",
}, location.origin);```

And then depending on that protocol value - filter only ergo or cardano wallets please

Reply:
In progress